pyazhpc/arm.py

```diff
@@ -689,7 +689,7 @@ def _add_vm(self, cfg, r, vnet_in_deployment):
                             "protocol": "Tcp",
                             "sourcePortRange": "*",
                             "destinationPortRange": "3389",
-                            "sourceAddressPrefix": "*",
+                            "sourceAddressPrefixes": "*",
```

Comment:
This is type "string[]" and so needs a list of strings.

Reply:
updated